### ISSUE TITLE
IsGift returns true when spapi returns "false"

### ISSUE THREAD

**Issue by haruny** [jlevers/selling-partner-api#165]
For some reason, when sp api returns:

"IsGift":"false", the getIsGift() returns true.

setDebug(true) returns the following payload:

`{"payload":{"OrderItems":[{"TaxCollection":{"Model":"MarketplaceFacilitator","ResponsibleParty":"Amazon Services, Inc."},"ProductInfo":{"NumberOfItems":"1"},"BuyerInfo":{},"ItemTax":{"CurrencyCode":"USD","Amount":"xx.00"},"QuantityShipped":1,"ItemPrice":{"CurrencyCode":"USD","Amount":"X.XX"},"ASIN":"XXXXX","SellerSKU":"xxxx-xxxx","Title":"*********","IsGift":"false","IsTransparency":false,"QuantityOrdered":1,"PromotionDisc
`
dump shows the following.
```
SellingPartnerApi\Model\Orders\GetOrderItemsResponse {#1372 ▼
  #container: array:3 [▼
    "payload" => SellingPartnerApi\Model\Orders\OrderItemsList {#1370 ▼
      #container: array:3 [▼
        "order_items" => array:1 [▼
          0 => SellingPartnerApi\Model\Orders\OrderItem {#1367 ▼
            #container: array:33 [▼
              "asin" => "xxxx"
              "seller_sku" => "xxxx"
              "order_item_id" => "xxxx"
              "title" => "xxxx"
              "quantity_ordered" => 1
              "quantity_shipped" => 1
              "product_info" => SellingPartnerApi\Model\Orders\ProductInfoDetail {#1364 ▶}
              "points_granted" => null
              "item_price" => SellingPartnerApi\Model\Orders\Money {#1355 ▶}
              "shipping_price" => null
              "item_tax" => SellingPartnerApi\Model\Orders\Money {#1352 ▶}
              "shipping_tax" => null
              "shipping_discount" => null
              "shipping_discount_tax" => null
              "promotion_discount" => SellingPartnerApi\Model\Orders\Money {#1347 ▶}
              "promotion_discount_tax" => SellingPartnerApi\Model\Orders\Money {#1387 ▶}
              "promotion_ids" => null
              "cod_fee" => null
              "cod_fee_discount" => null
              "is_gift" => true
              "condition_note" => null
              "condition_id" => null
              "condition_subtype_id" => null
              "scheduled_delivery_start_date" => null
              "scheduled_delivery_end_date" => null
              "price_designation" => null
              "tax_collection" => SellingPartnerApi\Model\Orders\TaxCollection {#1451 ▶}
              "serial_number_required" => null
              "is_transparency" => false
              "ioss_number" => null
              "store_chain_store_id" => null
              "deemed_reseller_category" => null
              "buyer_info" => SellingPartnerApi\Model\Orders\ItemBuyerInfo {#1452 ▶}
            ]
          }
        ]
        "next_token" => null
        "amazon_order_id" => "xxxx"
      ]
    }
    "errors" => null
    "headers" => array:8 [▶]
  ]
}
```

I have similar issue with getIsBusinessOrder() as well.

**Comment by jlevers** [comment]
Thanks for pointing this one out -- it was a deserialization issue (the string `"false"`) was being cast to `bool`, which resulted in the value `true`). Please upgrade to version `v4.2.5` and let me know if that fixes the issue for you.

**Comment by haruny** [comment]
I confirm, fixed in 4.2.5.
thank you for the quick fix!
